db/migrate/20210502113529_drop_view_bookmarks_permission.rb

```diff
@@ -0,0 +1,9 @@
+class DropViewBookmarksPermission < ActiveRecord::Migration[6.0]
+  def up
+    Permission.where(name: 'view_bookmarks').destroy_all
```

Comment:
This might result in empty Filters (if the `view_bookmarks` was the only permission granted by that filter), but I'd say that's very rare and quite harmless.

Reply:
Updated, thanks! actually realized my dev db had a bunch of empty filters left behind.